platforms/documentation/docs/src/docs/userguide/optimizing-performance/build-cache/caching_android_projects.adoc

```diff
@@ -51,6 +51,8 @@ When you next build, all of the outputs previously built while working on Branch
 
 == The Android Gradle Plugin and the Gradle Build Tool
```

Comment:
```suggestion
== The Android Gradle Plugin
```

Reply:
You can use this to add DCO.

Reply:
@Ayushcode10, go ahead and try to commit this with a sign off.

> To avoid having PRs blocked in the future, always include Signed-off-by: Author Name <authoremail@example.com> in every commit message. You can also do this automatically by using the -s flag (i.e., git commit -s).